hack/ova-parser/Containerfile

```diff
@@ -0,0 +1,10 @@
+FROM registry.access.redhat.com/ubi8/go-toolset:1.18.9-8 as builder
```

Comment:
It would be cleaner if we created a separate directory for the Container files naming them e.g. `ova-parser.Containerfile` and leave the `hack` directory only for support scripts.